group_vars/irods.yml

```diff
@@ -3,23 +3,39 @@ firewall_allowed_tcp_ports:                 # list of open ports on iCAT server
   - "22"    # SSH.
   - "443"   # davrods SSL
   - "1247"  # irods
+  - "1248"  # Control Plane Port
   - "5432"  # PostgreSQL
   - "20000:20199"  # irods
-irods_ssl_certificate_chain_file: "localhost_and_chain_umcg-icat01.crt"
-irods_ssl_certificate_key_file: "localhost-umcg01.key"
-irods_ssl_dh_params_file: "dhparams.pem"
-irods_zone: 'nlumcg'                        # default main iRODS zone name
-irods_local_resource: 'rootResc'            # local iRODS resource
-irods_vault_path: '/var/lib/irods/Vault'    # default path to store files for local resource
-irods_default_resource: 'surfObjStore'      # default resource iRODS uploads to
-irods_service_account: 'irods'              # linux account under which iRODS runs
-irods_admin_name: 'rods'                    # iRODS (and zone) account
-irods_admin_home_path: '/nlumcg/home/rods'  # iRODS admin's home path
-server_type: 'icat'                         # iRODS Server Type  
-irods_db_user: '{{ irods_service_account }}'  # db Username, usually same as irods_service_account
-irods_db_server: '127.0.0.1'                # iRODS Database Server
-irods_db_name: 'ICAT'                       # iRODS Database Name
-davrods_install: true                       # to install davrods docker
-davrods_docker_folder: "davrods_docker"     # davrods docker folder name, relative to the user home directory
-davrods_default_resource: "surfObjStore"    # default resource to upload files via davrods
+ir_server_type: 'icat'                  # iRODS Server Type
+ir_ssl_certificate_chain_file: 'localhost_and_chain_umcg-icat01.crt' # iRODS server certificate
+ir_ssl_certificate_key_file: 'localhost-umcg01.key' # iRODS server certificate's key
```

Comment:
These are currently located here: `files/nibbler_cluster/nemi_irods/` The last sub dir is specific for these machines, I would suggest:
```suggestion
ir_ssl_certificate_chain_file: 'nemi_irods/localhost_and_chain_umcg-icat01.crt' # iRODS server certificate
ir_ssl_certificate_key_file: 'nemi_irods/localhost-umcg01.key' # iRODS server certificate's key
```
which means the nemi_irods subfolder does not need to be hard code in the role and
```
- name: Install certificate key and chain files for iRODS
  ansible.builtin.copy:
    src: 'files/{{ stack_name }}/nemi_irods/{{ item.name }}'
    .....
```
can be changed into:
```
- name: Install certificate key and chain files for iRODS
  ansible.builtin.copy:
    src: 'files/{{ stack_name }}/{{ item.name }}'
    .....
```

Reply:
In addition some of the variables here are not "generic" for any of the iRODS servers we have or may want have in the future. E.g. `ir_zone`, `ir_default_res`, `ir_ssl_certificate_chain_file` and `ir_ssl_certificate_key_file` are rather specific for the current NEMI iRODS machines. Would be better to relocate those to the YAML-based static inventory.

Reply:
Agreed and changed. As the path for the certificate.